### DISCUSSION TITLE
Use of market data from diferent sources

### DISCUSSION THREAD

**Discussion by marineanalytica** [ssantoshp/Empyrial#67] [Q&A]
is there a way to use market data from different sources than yfinance? for instance, reading out data from a csv file on your computer?

**Comment by DannyBen** [comment]
I am also interested in this.

**Reply by santoshlite** [reply]
https://github.com/ssantoshp/Empyrial/releases/tag/v2.1.3

**Comment by santoshlite** [comment]
Coming soon

**Comment by santoshlite** [comment]
It exists in v2.1.3 now: https://github.com/ssantoshp/Empyrial/releases/tag/v2.1.3 took a while sorry :(
